### PR TITLE
infoschema: fix select * from inspection_summary would return null.

### PR DESCRIPTION
**Pull request by reafans**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:
1. Currently it would return empty results without rule condition.
### What is changed and how it works?
1. Make it return all the results for  `select * from inspection`.
2. Add a column name `COMMENT` to explain the row.

What's Changed:
before:
```
mysql> select * from inspection_summary;
Empty set (0.00 sec)
```

this pr :
```
mysql> select * from inspection_summary limit 10;
+-------+------------------+------------------------------------------+----------------+----------+-----------+-----------+-----------+------------------------------------------------------------------------------------------------------+
| RULE  | INSTANCE         | METRICS_NAME                             | LABEL          | QUANTILE | AVG_VALUE | MIN_VALUE | MAX_VALUE | COMMENT                                                                                              |
+-------+------------------+------------------------------------------+----------------+----------+-----------+-----------+-----------+------------------------------------------------------------------------------------------------------+
| stats | gncluster-tidb-0 | tidb_statistics_auto_analyze_duration    |                |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB auto analyze time durations within 95 percent histogram buckets                 |
| stats | gncluster-tidb-0 | tidb_statistics_fast_analyze_status      | access_regions |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB fast analyze statistics                                                         |
| stats | gncluster-tidb-0 | tidb_statistics_fast_analyze_status      | region_error   |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB fast analyze statistics                                                         |
| stats | gncluster-tidb-0 | tidb_statistics_fast_analyze_status      | sample         |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB fast analyze statistics                                                         |
| stats | gncluster-tidb-0 | tidb_statistics_fast_analyze_status      | scan_keys      |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB fast analyze statistics                                                         |
| stats | gncluster-tidb-0 | tidb_statistics_fast_analyze_status      | seek_keys      |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB fast analyze statistics                                                         |
| stats |                  | tidb_statistics_pseudo_estimation_ops    |                |     NULL |  0.000000 |  0.000000 |  0.000000 | TiDB optimizer using pseudo estimation counts                                                        |
| stats |                  | tidb_statistics_significant_feedback     |                |     NULL |  0.000000 |  0.000000 |  0.000000 | Counter of query feedback whose actual count is much different than calculated by current statistics |
| stats | gncluster-tidb-0 | tidb_statistics_stats_inaccuracy_rate    |                |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB statistics inaccurate rate                                                      |
| stats | gncluster-tidb-0 | tidb_statistics_store_query_feedback_qps | ok             |     NULL |  0.044445 |  0.044443 |  0.044445 | TiDB store quering feedback counts                                                                   |
+-------+------------------+------------------------------------------+----------------+----------+-----------+-----------+-----------+------------------------------------------------------------------------------------------------------+
10 rows in set (21.26 sec)

mysql> select * from inspection_summary where rule='query-summary' and metrics_name='tidb_query_duration';
+---------------+------------------+---------------------+----------+----------+-----------+-----------+-----------+----------------------------------------------+
| RULE          | INSTANCE         | METRICS_NAME        | LABEL    | QUANTILE | AVG_VALUE | MIN_VALUE | MAX_VALUE | COMMENT                                      |
+---------------+------------------+---------------------+----------+----------+-----------+-----------+-----------+----------------------------------------------+
| query-summary | gncluster-tidb-0 | tidb_query_duration | Begin    |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | Commit   |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | Delete   |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | Execute  |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | Insert   |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | Replace  |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | Rollback |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | Select   |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | Set      |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | Show     |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | Update   |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | Use      |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | general  |     0.99 |  0.000000 |  0.000000 |  0.000000 | The quantile of TiDB query durations(second) |
| query-summary | gncluster-tidb-0 | tidb_query_duration | internal |     0.99 |  0.004050 |  0.003934 |  0.005480 | The quantile of TiDB query durations(second) |
+---------------+------------------+---------------------+----------+----------+-----------+-----------+-----------+----------------------------------------------+
14 rows in set (0.09 sec)

```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:

### Check List 

Tests 

- Unit test
- Integration test

### CODE REVIEW

File: executor/inspection_summary.go [hunk 1]
```diff
@@ -422,8 +422,10 @@ func (e *inspectionSummaryRetriever) retrieve(ctx context.Context, sctx sessionc
 	condition := e.timeRange.Condition()
 	var finalRows [][]types.Datum
 	for rule, tables := range inspectionSummaryRules {
-		if !rules.exist(rule) {
-			continue
+		if len(rules.set) != 0 {
```
Comment (crazycs520) [inline executor/inspection_summary.go:425]: use `len(rules.set) != 0 && !rules.set.Exist(rule)`?

File: infoschema/tables.go [hunk 2]
```diff
@@ -950,6 +950,7 @@ var tableInspectionSummaryCols = []columnInfo{
 	{name: "AVG_VALUE", tp: mysql.TypeDouble, size: 22, decimal: 6},
 	{name: "MIN_VALUE", tp: mysql.TypeDouble, size: 22, decimal: 6},
 	{name: "MAX_VALUE", tp: mysql.TypeDouble, size: 22, decimal: 6},
+	{name: "COMMENT", tp: mysql.TypeVarchar, size: 64},
```
Comment (crazycs520) [inline infoschema/tables.go:953]: `size: 64` may be too short, use `256`, keep same with table `tableMetricSummaryCols `.

Comment (reafans) [reply infoschema/tables.go]: done

**Pull request merged**